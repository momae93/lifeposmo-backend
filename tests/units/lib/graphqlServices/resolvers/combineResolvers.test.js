const combineResolvers = require('../../../../../src/lib/graphqlServices/resolvers/combineResolvers');

describe('[LIB][GRAPHQL][RESOLVERS] : combineResolvers', () => {
  it('should match snapshot lib graphql combineResolvers', async () => {
    const mockUserServices = {};
    const mockpostServices = {};
    const mockServices = {
      mockUserServices,
      mockpostServices,
    };
    const resolvers = combineResolvers(mockServices);

    expect(resolvers).toMatchSnapshot();
  });
});
