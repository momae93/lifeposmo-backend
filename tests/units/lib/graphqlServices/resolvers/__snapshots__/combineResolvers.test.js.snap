// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[LIB][GRAPHQL][RESOLVERS] : combineResolvers should match snapshot lib graphql combineResolvers 1`] = `
Array [
  Object {
    "Mutation": Object {
      "createUser": [Function],
      "deleteUser": [Function],
      "followUser": [Function],
      "unfollowUser": [Function],
    },
    "Query": Object {
      "user": [Function],
      "users": [Function],
    },
    "User": Object {
      "postsWritten": [Function],
    },
  },
  Object {
    "Mutation": Object {
      "createPost": [Function],
      "deletePost": [Function],
    },
    "Post": Object {
      "author": [Function],
    },
    "Query": Object {
      "post": [Function],
      "posts": [Function],
      "postsWrittenBy": [Function],
    },
  },
  Object {
    "Date": "Date",
  },
]
`;
